osmosis-1/osmosis-frontier.assetlist.json

```diff
@@ -1395,6 +1395,34 @@
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/rac.svg"
       }
     },
+    {
+      "description": "Building Consensus among Hong Kong People",
+      "type_asset": "cw20",
+      "address": "juno1tdjwrqmnztn2j3sj2ln9xnyps5hs48q3ddwjrz7jpv6mskappjys5czd49",
+      "denom_units": [
+        {
+          "denom": "ibc/52E12CF5CA2BB903D84F5298B4BFD725D66CAB95E09AA4FC75B2904CA5485FEB",
+          "exponent": 0
+        },
+        {
+          "denom": "dhk",
+          "exponent": 0
+        }
+      ],
+      "base": "ibc/52E12CF5CA2BB903D84F5298B4BFD725D66CAB95E09AA4FC75B2904CA5485FEB",
+      "name": "DHK",
+      "display": "dhk",
+      "symbol": "DHK",
+      "ibc": {
+        "source_channel": "channel-47",
+        "dst_channel": "channel-169",
+        "source_denom": "dhk"
+      },
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/dhk.png",
```

Comment:
These should point to the images uploaded to the Cosmos Chain Registry. Please upload the images there with the registration so we can redirect these. Also, should make sure the images are the same--looks like the png has yellow text while the svg has transparent text.

Reply:
updated to url from cosmos chain registry and cosmos registry PR is awaiting as well